java/Makefile.am

```diff
@@ -25,9 +25,9 @@ CLEANFILES = generated-src/main/java/ola/proto/Ola.java \
              classes/ola/*.class \
              test-classes/ola/*.class \
              maven-archiver/pom.properties \
-	     surefire-reports/ola.*.txt \
-	     surefire-reports/TEST-*.xml \
-             ola-java-client-0.0.1.jar
+             surefire-reports/ola.*.txt \
```

Comment:
Can you fix the whitespace on these three lines, it should all be spaces, not tabs, so they're aligned equally with the lines above and below.